pkg/sdkserver/sdkserver_test.go

```diff
@@ -763,8 +758,6 @@ func TestSDKServerSendGameServerUpdate(t *testing.T) {
 
 	agruntime.FeatureTestMutex.Lock()
```

Comment:
You can remove all of the `FeatureTestMutex.Lock()` and `defer agruntime.FeatureTestMutex.Unlock()` lines since we are removing the feature gate.